### PR TITLE
Crud users

### PR DESCRIPTION
**Pull request by Letiste**

### CODE REVIEW

File: test/fixtures/users.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+foobar:
+  firstname: "Foo"
+  lastname: "Bar"
+  email: "foo@bar.com"
+  room: "A200"
```
Comment (nymous) [inline test/fixtures/users.yml:7]: I just had an idea about our fixtures: we could use fictional characters, that way we have something more memorable and easier to think about than "Foo Bar", but still without any personal information. Example:
```suggestion
ironman:
  firstname: "Tony"
  lastname: "Stark"
  email: "tony@avengers.com"
  room: "A200"
```

File: test/controllers/users_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        user: {
+          firstname: 'patrick',
+          lastname: 'bar',
+          email: 'patrickbar@bar.com',
+          room: 'E125'
+        }
+      }
+    end
+    user = @response.parsed_body
+    assert_template('show')
```
Comment (nymous) [inline test/controllers/users_controller_test.rb:54]: Is this 'show' or 'users/show', like in other tests?

**Review by nymous — CHANGES_REQUESTED** [review]
I think something went terribly wrong :rofl: 
image
Other than that, great first PR! It's a good beginning for this project :rocket:

File: README.md [hunk 3]
```diff
@@ -24,6 +24,7 @@ It is recommended to use [rbenv][rbenv] and [nvm][nvm] to install both language
    2. `npm install`
 5. Initialize the database (users, databases) by running [`init_db.sql`](.github/workflows/init_db.sql): `sudo --user postgres psql < ./.github/workflows/init_db.sql`
 6. (Optional) Edit [`config/database.yml`](config/database.yml) if you chose a different password
+7. Install [Overcommit](https://github.com/sds/overcommit): `overcommit --install`
```
[the remark this answers is not in the document]
Comment (Letiste) [reply README.md:27]: We can specify if you want. It works without adding `bundle exec`.

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        format.json { render 'show', status: :ok, location: @user }
+      else
+        format.html { render 'edit' }
+        format.json { render json: @user.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  def destroy
+    User.find(params[:id]).destroy
```
[the remark this answers is not in the document]
Comment (Letiste) [reply app/controllers/users_controller.rb:50]: If the user isn't found, it raises an error. In the dev env it is not handle and show the error on the page, to help debug. In the prod env, it renders a basic html page error (one from the public folder, depending on the error).

File: app/views/users/create.html.erb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+<h1>Users#create</h1>
```
[the remark this answers is not in the document]
Comment (Letiste) [reply app/views/users/create.html.erb:1]: You are right, the controller "action" doesn't need a template.

File: app/views/users/_user.html.erb [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<li>
+  <%= user.email %>,
```
[the remark this answers is not in the document]
Comment (Letiste) [reply app/views/users/_user.html.erb:2]: It is only a temporary template, the name is rendered when all the users are listed if you want to check it.

File: test/fixtures/users.yml (hunk 1 above)
Comment (Letiste) [reply test/fixtures/users.yml:7]: I like this idea but i hope you are not suggesting this because you are sponsored by Disney

**Review by nymous — APPROVED** [review]

**Pull request merged**